### PR TITLE
Add a new make target: new-pool-admins

### PR DESCRIPTION
**Pull request by hongkailiu**
Follow up https://github.com/openshift/ci-docs/pull/165

/cc @petr-muller 

I will clean up `group/cluster-namespace-viewer` after merge.

### CODE REVIEW

File: clusters/hive/pools/_pool-admin-rbac_template.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  kind: Group
+  metadata:
+    name: "${TEAM}-pool-admins"
+  users:
+  - "${ADMIN}"
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: "${TEAM}-pool-admins"
+    namespace: hypershift-cluster-pool
```
Comment (hongkailiu) [inline clusters/hive/pools/_pool-admin-rbac_template.yaml:16]: This is wrong.

Comment (hongkailiu) [reply clusters/hive/pools/_pool-admin-rbac_template.yaml]: Corrected.

File: hack/generate_new_pool_admins.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [[ "$#" -ne 2 ]]; then
+    echo "Illegal number of parameters"
```
Comment (stevekuznetsov) [inline hack/generate_new_pool_admins.sh:6]: print usage so the user knows what is expected

File: hack/generate_new_pool_admins.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    exit 1
+fi
+
+
+TEAM=$1
+OWNERS=$2
+IFS=',' read -r -a OWNERS_ARRAY <<< "${OWNERS}"
+
+OWNERS_PARAM=$(printf ",\"%s\"" "${OWNERS_ARRAY[@]}")
```
Comment (stevekuznetsov) [inline hack/generate_new_pool_admins.sh:15]: you need `%q` and likely want to merge this using `$IFS` or an inline Python script, please - this sort of Bash is very hard to maintain and catch edge cases

**Review by stevekuznetsov — APPROVED** [review]

**Comment by hongkailiu** [comment]
```
oc --context hive delete group cluster-namespace-viewer
group.user.openshift.io "cluster-namespace-viewer" deleted

oc --context hive delete ClusterRoleBinding cluster-namespace-view
clusterrolebinding.rbac.authorization.k8s.io "cluster-namespace-view" deleted
```

**Pull request merged**